app/controllers/hiring_staff/vacancies/documents_controller.rb

```diff
@@ -26,4 +30,68 @@ def redirect_if_no_supporting_documents
   def redirect_to_next_step_if_save_and_continue
     redirect_to application_details_school_job_path if params[:commit] == 'Save and continue'
   end
+
+  def add_documents(documents_attributes)
```

Comment:
Could you hoist this into the create action?  There's no real reason to have a four-line function that only gets called once.

Reply:
I only did this because Rubocop checks were failing. Happy to refactor however makes most sense.